.github/actions/spell-check/only.txt

```diff
@@ -2,3 +2,4 @@
 ^docs/
 \.md$
 pdns/recursordist/settings/table.py
+regression-tests.*/description
```

Comment:
As noted, this was moved from `excludes.txt` and was a valuable move -- also, this change in concert with the check-file-names feature is what caught the directory changes below...